### DISCUSSION TITLE
dashPanel menu issue

### DISCUSSION THREAD

**Discussion by mhmatttu** [tagyoureit/nodejs-poolController#773] [General]
I am unable to open the "hamburger" menu in dashPanel in Google chrome.  Works just fine in Edge browser on same computer or from my phone.  I have uninstalled and reinstalled Chrome but same result.  Below is the error in Console.  Thoughts?

![image](https://github.com/tagyoureit/nodejs-poolController/assets/47801991/74e282ec-fa45-409e-ba4c-d6cc2f79c184)

**Comment by rstrouse** [comment]
Pull dashPanel this was google doing as much harm as possible per their mission statement.  The popovers needed to have the 0 margins or they would not open since the chrome update.

**Reply by mhmatttu** [reply]
working great. thank you!

**Comment by mhmatttu** [comment]
The update to dashPanel worked great for my system but I updated my BIL’s system and still having the popover issue. Below shows that both njsPC and dashPanel are current. I did have an issue with PM2 when updating node to version 18 but got it running again by reinstalling PM2.  I saw another post about deleting the node_module directory and re-running npm i which I did for njsPC and dashPanel. Any ideas?

![IMG_1767](https://github.com/tagyoureit/nodejs-poolController/assets/47801991/a0e9031e-cd97-49e1-b558-e7ee5c4402a8)

**Reply by rstrouse** [reply]
Please hard refresh your browser or clear the browser cache.

**Comment by triorbonk** [comment]
I am seeing the same problem but am unable to see the errors reported by mhmatttu.  I assume this is due to an update in chrome engine as I see the same problem on both mobile and browser as well as edge browser.  
I am able to reproduce this in incognito mode with chrome, so I don't believe it is a cookie or session state issue.

**Reply by rstrouse** [reply]
It is not a session state issue it is a cache issue.  Please pull the latest from dashPanel
`cd ~/nodejs-poolController-dashPanel`
`git pull`
-- Double check to ensure it actually pulled and did not abort for existing changes if it did run `git reset --hard`
`npm i`

Then hard refresh your browser.

**Reply by tagyoureit** [reply]
FYI, on Chrome mobile there isn't a "hard refresh" per web page.  You need to go and clear your entire cache history.  Alternatively, I would first try a different browser (Safari, GoGoDuck, Firefox, etc) and see if the problem persists outside of Chrome.

**Reply by triorbonk** [reply]
Upgrade fixed it for me.  Thanks.  

Eric
